### PR TITLE
fix: ui template for auth UI-1058

### PR DESCRIPTION
**Pull request by ekdev2**

### CODE REVIEW

File: templates/ui-package/src/app/app.component.ts [hunk 1]
```diff
@@ -9,9 +9,21 @@ import {filter} from 'rxjs/operators';
...
+    this.onAuthorizationResult = this.authService.onAuthorizationResult();
+  }
   title = '<%= appNameSlug %>';
   ngOnInit(): void {
+    this.authService.onAuthCallback();
+
+    // Observable for receiving the auth's events
+    this.onAuthorizationResult.subscribe(result => {
+      if (result.authorizationState === 'authorized') {
+        console.log(`authorized`);
```
Comment (rafaelcalpena) [inline templates/ui-package/src/app/app.component.ts:23]: Should we remove the console.log?

File: templates/ui-package/src/app/auth/auth.component.ts [hunk 2]
```diff
@@ -29,18 +26,12 @@ export class AuthComponent implements OnInit {
...
-        console.log(\`authorized\`);
-
-      }
-    });
-    //Observable to determine if is authorized or not
+    // Observable to determine if is authorized or not, default value is undefined
     this.authService.isAuthorized().subscribe(authorized => {
-      this.isLoggingEnable =authorized;
-      console.log('is authorized: ' + authorized);
+      if(authorized || authorized === false) {
```
Comment (rafaelcalpena) [inline templates/ui-package/src/app/auth/auth.component.ts:31]: What are the possibilities for authorized variable?

File: templates/ui-package/src/app/auth/auth.component.ts [hunk 3]
```diff
@@ -29,18 +26,12 @@ export class AuthComponent implements OnInit {
...
+        this.isLoggingEnable = authorized;
+        console.log('is authorized: ' + authorized);
```
Comment (rafaelcalpena) [inline templates/ui-package/src/app/auth/auth.component.ts:33]: console.log

File: templates/ui-package/src/app/auth/auth.component.ts [hunk 4]
```diff
@@ -53,34 +44,38 @@ export class AuthComponent implements OnInit {
...
 
-  ngOnInit() {
-    // Observable for receiving the auth's events, also you can use the default behaviour
-    // which is redirect
-    this.authService.onAuthorizationResult().subscribe(result => {
+    // Observable for receiving the auth's events
+    this.onAuthorizationResult.subscribe(result => {
       if (result.authorizationState === 'authorized') {
-        console.log(`authorized`);
+        console.log('authorized');
```
Comment (rafaelcalpena) [inline templates/ui-package/src/app/auth/auth.component.ts:62]: console.log

File: templates/ui-package/src/app/auth/auth.component.ts (hunk 2 above)
Comment (JamilOmar) [reply templates/ui-package/src/app/auth/auth.component.ts]: true  false or undefined

**Pull request merged**